ops/charm.py

```diff
@@ -1005,7 +1005,7 @@ class CharmBase(Object):
 
     :code:`CharmBase` is used to create a charm. This is done by inheriting
     from :code:`CharmBase` and customising the subclass as required. So to
-    create your own charm, say ``MyCharm``, define a charm class and set up the
+    create a charm, say ``MyCharm``, define a charm class and set up the
```

Comment:
Not your issue, but "say" reads strangely to me. Perhaps:
```suggestion
    create a charm called ``MyCharm``, define a charm class and set up the
```